ci/github/helpers/install_rclone.bash

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+#  Installs the latest version of rclone plugin
+#
+
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -o errexit   # abort on nonzero exitstatus
+set -o nounset   # abort on unbound variable
+set -o pipefail  # don't hide errors within pipes
+IFS=$'\n\t'
+
+
+# Installation instructions from https://rclone.org/docker/
+
+apt-get -y install fuse
+mkdir -p /var/lib/docker-plugins/rclone/config
+mkdir -p /var/lib/docker-plugins/rclone/cache
+docker plugin install rclone/docker-volume-rclone:amd64 args="-v" --alias rclone --grant-all-permissions
```

Comment:
same here right? amd64 will always point to the latest. should we not have the same versions as in the deploys?
and maybe have a test for the newer versions?

Reply:
I did not pin a version, we may do that. I installed the latest version. As before, we may pin a version. Let me know which, I will adjust the ops-ansible playbooks accordingly.